lib/node-integration-iframe.mustache.js

```diff
@@ -1,4 +1,8 @@
-(function () {
+(function (runtimeFlag) {
+  if (window[runtimeFlag] === true) {
```

Comment:
FWIW, I was using that flag along with `useIframe: false` which puts Karma at the very foreground of the Electron container but it wasn't worth the trouble in the end as I had to clean up after webpack entries too.

Thanks for the suggestion.